src/components/UserAvatar/index.js

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
deveria funcionar sem importar o React explicitamente. Tem alguma razão pra precisar disso?

Reply:
Solução para o erro `'React' must be in scope when using JSX  react/react-in-jsx-scope`.
Vou verificar atualizações para ver se a exigência persiste.